### ISSUE TITLE
Some doubts about config.py

### ISSUE THREAD

**Issue by dagongji10** [zhang0jhon/AttentionOCR#78]
@zhang0jhon Thanks for sharing a nice work!
I plan to train my own datasets with AttentionOCR and when I modify `config.py`, I got some doubts:
1. `restore_path`: how to use this para? Set it to /path/to/pretrained.ckpt? Need I change `starting_epoch`?
2. `steps_per_epoch`: I only have 2 GPUs, so set `batch_size=5`, and how to change `steps_per_epoch`? My datasets have 56030 samples, must I set `steps_per_epoch=56030/5`?
3. `warmup_steps`: how to use this para? Must it keep same with `num_epochs`?

**Comment by alan-img** [comment]
> @zhang0jhon Thanks for sharing a nice work!
> I plan to train my own datasets with AttentionOCR and when I modify `config.py`, I got some doubts:
> 
> 1. `restore_path`: how to use this para? Set it to /path/to/pretrained.ckpt? Need I change `starting_epoch`?
> 2. `steps_per_epoch`: I only have 2 GPUs, so set `batch_size=5`, and how to change `steps_per_epoch`? My datasets have 56030 samples, must I set `steps_per_epoch=56030/5`?
> 3. `warmup_steps`: how to use this para? Must it keep same with `num_epochs`?

your problem I don't know, but I want to know what is label tool of labeling you own datset. because I also want to train in my own dataset.
